frontend/www/js/omegaup/components/arena/EphemeralGrader.vue

```diff
@@ -21,8 +21,22 @@ export default class EphemeralGrader extends Vue {
 
   mounted(): void {
     (this.$refs.grader as HTMLIFrameElement).onload = () => this.iframeLoaded();
+
+    window.addEventListener('resize', this.handleWindowResize);
+  }
+  unmounted(): void {
+    window.removeEventListener('resize', this.handleWindowResize);
   }
+  handleWindowResize(): void {
+    const iframe = this.$refs.grader as HTMLIFrameElement;
 
+    // zoom in/out manually using device pixel ratio
+    // zoom property was recently supported on firefox browser
+    // so this approach works on most browsers
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    iframe.style.zoom = 1 / window.devicePixelRatio;
```

Comment:
Will it be possible to avoid the comment that ignores the TypeScript error?

If the problem is due to zoom not being available for all browsers, you can add an interface:

```ts

interface CSSStyleDeclaration {
  zoom?: string | number;
}
```

Additionally, you can indicate it here as follows:
```suggestion
    (iframe.style as CSSStyleDeclaration).zoom = String(
      1 / window.devicePixelRatio,
    );
```

Reply:
Yes this makes sense.